newIDE/app/scripts/extract-changelog.js

```diff
@@ -201,4 +204,13 @@ const formatHiddenCommitMessage = commit => {
 
   shell.echo(`\n## 🐛 Bug fixes\n`);
   shell.echo(fixCommits.map(formatCommitMessage).join('\n'));
+
+  if (devCommits) {
+    shell.echo(`\n<details>`);
+    shell.echo(`<summary> 🛠 Developers</summary>`);
```

Comment:
You should not use HTML as if I remember correctly the changelog is extracted to be put on GitHub. GitHub will escape the HTML for obvious security reasons, and there will just be the ugly tags polluting the changelog without their effect. Just use markdown ;)

Reply:
This HTML work in GDevelop and github :)

Reply:
It seems so, nevermind about the fact that GitHub escapes it, but it still is a security issue. Imagine someone makes a MitM attack and adds malicious scripts to the changelog, it wouldn't be escaped by GDevelop and give remote code execution on an environment with node, that is very dangerous!
Maybe GDevelop could replace those specific HTML tags with a Markdown version and use a plugin for the markdown parser to add spoilers.

Reply:
Here is a remark (the used markdown renderer) plugin that would do that: https://github.com/Rokt33r/remark-collapse

Reply:
We should indeed not accept arbitrary html. Let me know if you can use this or something else to accept **only** summary and details :)